packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -1321,6 +1322,49 @@ export enum FileType {
     SymbolicLink = 64
 }
 
+/**
+ * The `FileStat`-type represents metadata about a file
+ */
+export interface FileStat {
```

Comment:
I could not find anthing like that in https://github.com/microsoft/vscode/blob/master/src/vs/workbench/api/common/extHostTypes.ts

We should align these files. It should contain implementation of VS Code types.

Reply:
leftover, not using filestat and as it's an interface, don't need it